junit/kube-api-test/core/src/test/java/io/fabric8/kubeapitest/sample/JUnitExtensionOnMethodTest.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.fabric8.kubeapitest.sample;
+
+import io.fabric8.kubeapitest.junit.EnableKubeAPIServer;
+import io.fabric8.kubeapitest.junit.KubeConfig;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Shows usage for the case when fresh Kube API is started for each use test method.
+ */
+class JUnitExtensionOnMethodTest {
+
+  // not static in case if API server is restarted for each test
+  @KubeConfig
+  String kubeConfigYaml;
+
+  @Test
+  @EnableKubeAPIServer
+  void simpleTest1() {
+    TestCaseUtils.simpleTest(kubeConfigYaml);
+  }
+
+  @Test
+  @EnableKubeAPIServer
+  void simpleTest2() {
+    TestCaseUtils.simpleTest(kubeConfigYaml);
```

Comment:
Sorry, I haven't checked deeply but on a quick look I'm not able to figure out how `simpleTest1()` is testing something different than `simpleTest2()` . Or could it be a typo?

Reply:
If we want to deliberately repeat the execution, would[ `@RepeatedTest](https://junit.org/junit5/docs/5.4.1/api/org/junit/jupiter/api/RepeatedTest.html)` work in this case?

Reply:
I think the intention was here to test if this whole stuff works with the repeated annotation, but it looks weird to me a bit, will revisit this too later if no objections.